wiki/Client/Beatmap_editor/Compose/en.md

```diff
@@ -22,6 +22,8 @@ The editor shares the same tools between osu!, osu!taiko, and osu!catch, while o
 
 The timeline can be zoomed in and out with the `+`/`-` buttons to the left, or alternatively scrolling with the `Alt` key held. The two white lines in the middle indicate the current timestamp. Additionally, hit objects on the timeline can be selected and moved around with the left mouse button, or removed by right-clicking.
 
+Click and drag a slidertail on the timeline right to create [reverse sliders](/wiki/Gameplay/Hit_object/Slider/Reverse_slider).
```

Comment:
Where did the term "reverse slider" come from? "repeat slider" seems more fitting, no?

Reply:
Probably because the arrow is called a "reverse arrow" in the [new beginnings](https://osu.ppy.sh/beatmapsets/1011011#osu/2116202) tutorial beatmap and the [skinning file name](https://osu.ppy.sh/wiki/en/Skinning/osu%21#slider):

<img width="415" alt="Screenshot 2024-11-13 at 10 13 55 PM" src="https://github.com/user-attachments/assets/de7d00f0-d51d-4958-8600-bdfa5993e6d4">
<img width="413" alt="Screenshot 2024-11-13 at 10 15 34 PM" src="https://github.com/user-attachments/assets/c77933e8-d242-473d-9347-1035cf52a04c">

There are also more cases of "reverse arrow/slider" than "repeat arrow/slider" in the wiki.

Reply:
based on this, I think I should replace "create reverse sliders" with "add reverse arrows"

Though it possibly makes things temporarily inconsistent. So I guess this pr should also rename the "reverse slider" page + a few references?

Reply:
yup, renaming it, with adding a redirect from the previous name, would be appreciated

Reply:
did just that in 3c171be + 59462b5 -- hope it's not too hasty